### DISCUSSION TITLE
deviation of RT from Predicted.RT - metric of LC quality?

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#705] [Q&A]
1) I presume deviation of RT from Predicted.RT is a relevant metric of LC quality. Correct? In DIA it is a challenge if this deviation gets too large, as library matching then becomes a challenge, correct?

2) Along these lines: I presume Predicted.RT is the expected RT for a given peptide/precursor/fragment etc. when correcting for shifts in that sample - correct? In that sense, the deviation of RT from Predicted.RT is a residual error after a correction, correct? Just that the expectation gets projected onto the real sample RT ('Predicted.RT'), instead of the real sample RT getting projected (could be called "corrected RT") onto the expected RT (e.g. library ref).

**Comment by vdemichev** [comment]
1. It reflects how similar is the type of chromatography to the one used to make the library. DIA-NN will be fine on most samples without RT information at all, but IDs of course will drop.
2. Yes. Exactly.
